rubygem-foreman_maintain/rubygem-foreman_maintain.spec

```diff
@@ -2,13 +2,13 @@
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name foreman_maintain
-%global confdir foreman_maintain
+%global package_name foreman-maintain
```

Comment:
Thanks @ekohl for the suggestion. I've renamed `package_name` to `directory_name`.